docs/config_apppwd.rst

```diff
@@ -0,0 +1,133 @@
+.. _config_apppwd:
+
+App Passwords
+=============
+
+Background
+----------
+
+This Feature enables you to configure individiual Application Passwords that are 
```

Comment:
The feature should be this generic, as far as I can tell. If you are in need for a custom password field, the password is needed for some application, for example WLAN. You want to store the hash in the field "wlanPassword", with an NTLM hash. On the website, you want the menu to include a button "Change password for WLAN".
This results in:
```php
   $change_apppwd = array(
      array(
        'attribute' => "wlanPassword",
        'hash' => "NTLM",
        'label' => "WLAN"
      )
   )
```

My PR does not provide an application called password. It is meant for changing passwords of certain applications.

Reply:
Actually, your PR allows to change a custom LDAP field of any object.

In your use case, you name the object an "application", but it could be any other object, including a user, an organization,...

Whether this field (which turns to be a password) is used by 0, 1, or more applications, SSP or LDAP should not be aware of. (moreover it can change with time)

Reply:
replaced apppwd with custompwdfield, in code and documentation.

These were some groundbreaking changes, please give me some time testing everything if everything is still working as expected.